src/server/pfs/server/testing/server_test.go

```diff
@@ -5201,6 +5201,7 @@ func TestPFS(suite *testing.T) {
 		c := env.PachClient
```

Comment:
Should we set up a test environment per test?  I notice these occasionally break.  Is that because they share the same test environment?

Reply:
Yeah, I'll make that change.